server/src/serializers/SandwichSerializer.js

```diff
@@ -26,7 +26,10 @@ class SandwichSerializer {
       serializedSandwich[field] = sandwich[field]
     })
     const relatedReviews = await sandwich.$relatedQuery("reviews")
-    serializedSandwich.reviews = await ReviewSerializer.summaryForShow(relatedReviews)
+    serializedSandwich.reviews = await ReviewSerializer.summaryForShow(
+      relatedReviews,
+      userId
+    )
```

Comment:
unless this is over 80 characters, I dont see much value in making this addition a multiline